js/widgets/forms/clearButton.js

```diff
@@ -53,7 +64,7 @@ define( [
 			}
 
 			$.extend( this, {
-				_clearBtn: this.widget().find("a.ui-input-clear")
+				_clearBtn: this.widget().find( "a.ui-textinput-clear" )
```

Comment:
Within this extension `this.widget()` is always going to be the wrapper. In the main textinput we should be setting `this.widgetElement` ( or similar ) during create since it cant change. Then just reference that in the `widget` method and here.